Resources/Prototypes/Entities/Mobs/NPCs/animals.yml

```diff
@@ -232,6 +232,120 @@
   - type: TimedDespawn #delete the egg after the chicken spawns
     lifetime: 21
 
+- type: entity # TODO: figure out how to make these guys gib when stepped on
```

Comment:
steptrigger then make gibontrigger it will be funny

Reply:
whatever i do i cant make a mob activate steptrigger, im just gonna resolve this and leave that for another pr

Reply:
rip probably some fixture chicanery

Reply:
i even tried to replace the entire fixture with the mousetrap fixture which i know works but it still doesnt work :mistake: